modules/dnn/test/test_layers_1d.cpp

```diff
@@ -12,41 +12,948 @@
+        inp_value = 1 + static_cast <float> (rand()) / (static_cast <float> (RAND_MAX/(10-1)));
+        input = cv::Mat(input_shape, CV_32F, inp_value);
+    }
+};
+
 TEST_P(Layer_1d_Test, Scale)
 {
-    int batch_size = get<0>(GetParam());
+    LayerParams lp;
+    lp.type = "Scale";
+    lp.name = "scaleLayer";
+    lp.set("axis", 0);
+    lp.set("mode", "scale");
+    lp.set("bias_term", false);
+    Ptr<ScaleLayer> layer = ScaleLayer::create(lp);
+
+    input = cv::Mat(input_shape, CV_32F);
+    cv::randn(input, 0.0, 1.0);
+    cv::Mat weight = cv::Mat(output_shape, CV_32F, 2.0);
+
+    std::vector<Mat> inputs{input, weight};
+    std::vector<Mat> outputs;
+
+    cv::Mat output_ref = input.mul(weight);
+    runLayer(layer, inputs, outputs);
+
+    ASSERT_EQ(shape(output_ref), shape(outputs[0]));
+    normAssert(output_ref, outputs[0]);
+}
+
+TEST_P(Layer_1d_Test, ReLU6)
+{
+    LayerParams lp;
+
+    lp.type = "ReLU6";
+    lp.name = "ReLU6Layer";
+    lp.set("min_value", 0.0);
+    lp.set("max_value", 1.0);
+    Ptr<ReLU6Layer> layer = ReLU6Layer::create(lp);
+
+    cv::Mat output_ref(output_shape, CV_32F, 1.0);
+    std::vector<Mat> inputs{input};
+    std::vector<Mat> outputs;
+
+    runLayer(layer, inputs, outputs);
+    ASSERT_EQ(shape(output_ref), shape(outputs[0]));
+    normAssert(output_ref, outputs[0]);
+}
+
+TEST_P(Layer_1d_Test, Clip)
+{
+    LayerParams lp;
+
+    lp.type = "Clip";
+    lp.name = "clipLayer";
+    lp.set("min_value", 0.0);
+    lp.set("max_value", 1.0);
+    Ptr<ReLU6Layer> layer = ReLU6Layer::create(lp);
+
+    cv::Mat output_ref(output_shape, CV_32F, 1.0);
+    std::vector<Mat> inputs{input};
+    std::vector<Mat> outputs;
+
+    runLayer(layer, inputs, outputs);
+    ASSERT_EQ(shape(output_ref), shape(outputs[0]));
+    normAssert(output_ref, outputs[0]);
+}
+
+TEST_P(Layer_1d_Test, ReLU)
+{
+    LayerParams lp;
+
+    lp.type = "ReLU";
+    lp.name = "reluLayer";
+    lp.set("negative_slope", 0.0);
+    Ptr<ReLULayer> layer = ReLULayer::create(lp);
+
+    cv::Mat output_ref(output_shape, CV_32F, inp_value);
+    std::vector<Mat> inputs{input};
+    std::vector<Mat> outputs;
+
+    runLayer(layer, inputs, outputs);
+    ASSERT_EQ(shape(output_ref), shape(outputs[0]));
+    normAssert(output_ref, outputs[0]);
+}
+
+TEST_P(Layer_1d_Test, Gelu)
+{
+    LayerParams lp;
+
+    lp.type = "Gelu";
+    lp.name = "geluLayer";
+    Ptr<GeluLayer> layer = GeluLayer::create(lp);
+
+    float geluValue = inp_value * 0.5 * (std::erf(inp_value * 1 / std::sqrt(2.0)) + 1.0);
+
+    float data[3] = {geluValue, geluValue, geluValue};
+    cv::Mat output_ref(output_shape, CV_32F, data);
+    std::vector<Mat> inputs{input};
+    std::vector<Mat> outputs;
+
+    runLayer(layer, inputs, outputs);
+    ASSERT_EQ(shape(output_ref), shape(outputs[0]));
+    normAssert(output_ref, outputs[0]);
+}
+
+TEST_P(Layer_1d_Test, GeluApprox)
+{
+    LayerParams lp;
+
+    lp.type = "GeluApprox";
```

Comment:
Please be accurate following layer factory https://github.com/opencv/opencv/blob/4.x/modules/dnn/src/init.cpp. If we use `LayerParams` with `Net` to create a net with single layer, `lp.type` needs to match one of the keys in the layer factory.